### ISSUE TITLE
Clones are not torn down in Ember 2.3.2

### ISSUE THREAD

**Issue by mellatone** [peec/ember-css-transitions#11]
When using either the mixin or the component version, the teardown removes the component but not the DOM clone, leaving a copy behind each time something like a toggle happens.

**Comment by mellatone** [comment]
![image](https://cloud.githubusercontent.com/assets/3988782/14477436/b8c76934-00c2-11e6-84af-2d70e2673f5f.png)

**Comment by milindalvares** [comment]
👍 Facing the same issue.

**Comment by miguelcobain** [comment]
I released now 0.1.5.
The strategy has changed and we shouldn't have these kind of problems anymore. See #15

**Comment by fedekau** [comment]
@miguelcobain This is still happening I am using ember-paper with the last alpha version and it still happening with the paper-dialog component, I have tried to find where/how the problem originates but I haven't yet, do you have any idea where it could be? If you give me a hint maybe I could help you solve that!

**Comment by fedekau** [comment]
I think I found the problem, ember-wormhole uses two text nodes to identify what elements to delete, but the cloned element is inserted outside those elements created by the wormhole addon and that might be why they are not being deleted, just an hypothesis I need to prove it now! 🤓

**Comment by fedekau** [comment]
This is what happens now, the last element is the cloned one, but it should not be! The last element should be the text node in position 7 which was created by the wormhole, and that is the problem (I hope)

<img width="701" alt="screen shot 2016-11-23 at 8 25 39 pm" src="https://cloud.githubusercontent.com/assets/7522836/20581974/665a696c-b1bb-11e6-8a46-8108b2d3d686.png">

**Comment by miguelcobain** [comment]
@fedekau does this happen in all browsers?

**Comment by fedekau** [comment]
@miguelcobain It happens on Chrome and Firefox, in Safari I couldn't reproduce the problem. I am using the last stable version of each browser.

**Comment by miguelcobain** [comment]
@fedekau Very strange. This test is passing on Chrome for me: https://github.com/peec/ember-css-transitions/blob/master/tests/integration/components/transition-group-test.js#L48

Do you see that bug in latest ember-paper's docs?

**Comment by fedekau** [comment]
@miguelcobain It don't see the bug there, I will try to update an example where it fails so you can see it! Thanks for the time :)

**Comment by miguelcobain** [comment]
A failing test or some way of reproducing the bug would help immensely.

**Comment by fedekau** [comment]
@miguelcobain I took a look at the problem again and I am not sure if what I previously said is valid (about the clone not being deleted), anyway here is an example app [ember-css-transitions-bug](https://github.com/fedekau/ember-css-transitions-bug)

To reproduce it:

1. Click the `more_vert` icon ( three dots)
2. Click the link `Route 1`
3. Navigate back with the browser back arrow (this should not reload the app, if it does you will not see the error)
4. If you navigate forward with the browser arrow
5. Repeat from 3

This will add an element to the document every time you do the steps 3 & 4

Let me know if you can see the problem, and if you need help to fix it!

**Comment by miguelcobain** [comment]
@fedekau The problem isn't in ember-css-transitions.
Ember-paper has it's own mixin for 3d translation on dialogs.
In that mixin we do our own cloning and destruction logic, like in ember-css-transitions.

ember-css-transitions once had this problem, but then I fixed it in https://github.com/peec/ember-css-transitions/pull/15.
We need to do the same thing in ember-paper's translate3d mixin.

In fact, I kept note that this bug might happen in https://github.com/miguelcobain/ember-paper/issues/249:
> don't use transitionend events for animation. Figure out a way to reuse ember-css-transitions "wait for animations" logic.

tl;dr transitionend events are useless and buggy.

**Comment by fedekau** [comment]
Thanks for taking the time to explain that @miguelcobain!
